RedHatInsights/patchman-ui
# fix(SOM-2082): properly update URL params in useGetEntities hook

# Description

Associated Jira ticket: # [(issue)](https://issues.redhat.com/browse/SPM-2082)

Fixes the missing global filter chip after page refresh on Systems and advisory systems pages and also removes redundant trash code in Hooks.js


# How to test the PR

1. Navigate to the system all page.
2. Select SAP from the global filter.
3. Refresh the page
4. Check workloads SAP filter after refreshing the page

# Before the change


# After the change


# Dependent work link


# Checklist:
